integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -828,21 +829,42 @@ public void createDomainOnExistingDirectory() throws Exception {
             + "/domains/"
             + domainMap.get("domainUID")
             + "\"";
-    logger.info("making sure the domain directory exists by running " + cmd);
-    ExecResult result = TestUtils.exec(cmd);
-    // logger.info("Command result " + result.stdout() + " err =" + result.stderr());
-    logger.info("Run the script to create domain");
-
-    // create domain using different output dir but pv is same, it fails as the domain was already
-    // created on the pv dir
-    try {
-      callCreateDomainScript(userProjectsDir + "2");
-    } catch (RuntimeException re) {
-      re.printStackTrace();
-      logger.info("[SUCCESS] create domain job failed, this is the expected behavior");
-      return;
+      logger.info("making sure the domain directory exists by running " + cmd);
+      ExecResult result = TestUtils.exec(cmd);
+      // logger.info("Command result " + result.stdout() + " err =" + result.stderr());
```

Comment:
else part can start here as rest of the code is same in both conditions..? 

this is same in both conditions ...
```
logger.info("Run the script to create domain");

      // create domain using different output dir but pv is same, it fails as the domain was already
      // created on the pv dir
      try {
        callCreateDomainScript(userProjectsDir + "2");
      } catch (RuntimeException re) {
        re.printStackTrace();
        logger.info("[SUCCESS] create domain job failed, this is the expected behavior");
        return;
      }
throw new RuntimeException("FAIL: unexpected result, create domain job did not report error");
```